lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
+from StringIO import StringIO
+
+
+class TestUsernameRenameCommand(unittest.TestCase):
+    '''
+    Class which contains various unit tests for the manage.py rename_user command line tool
+    '''
+    @classmethod
```

Comment:
I was having issues seeding data with the instance method setUp because of a DuplicateKey error in mongo. I solved this by changing the external_id field of the user to be unique.